src/DashboardService.Business/Board/GetBoardsCommand.cs

```diff
@@ -25,9 +29,22 @@ public class GetBoardsCommand : IGetBoardsCommand
     _responseCreator = responseCreator;
   }
 
-
-  public Task<FindResultResponse<BoardInfo>> ExecuteAsync(GetBoardsFilter filter)
+  public async Task<FindResultResponse<BoardInfo>> ExecuteAsync(GetBoardsFilter filter, CancellationToken ct)
   {
-    throw new System.NotImplementedException();
+    (List<DbBoard> boards, int totalCount) = await _boardRepository.GetAllAsync(filter, ct);
+
+    if (boards is null || !boards.Any())
+    {
+      return new()
+      {
+        Body = new List<BoardInfo>(),
+      };
+    }
+
+    return new()
+    {
+      Body = boards.Select(x => _boardInfoMapper.Map(x)).ToList(),
```

Comment:
x => _boardInfoMapper.Map(x) to _boardInfoMapper.Map and convertAll

Reply:
++